wiki/Handling_Cheating/pt-br.md

```diff
@@ -0,0 +1,75 @@
+Por que fui banido?
+----------------
+
+-   10 silêncios emitidos na sua página de usuário.
+-   Você deu a sua conta para outra pessoa.
+-   Usou várias contas.
+-   Trapaceou (veja abaixo).
+
+O que é considerado trapaça?
+----------------------------
+
+Como as regras dizem, qualquer forma de manipulação do cliente; hack de pontuação, macro em sequencias de combo e mira automática são todas consideradas trapaça.
+
+Isso é uma regra de um golpe; trapaceiros serão **banidos permanentemente.**
+
+Fui banido por trapacear mas eu não usei trapaça!
+------------------------------------------------
+
+O osu! usa métricas automatizadas combinadas com confirmação manual.
+
+A equipe nunca bane alguém incorretamente por trapacear. Antes que eles façam qualquer ação, eles se certificam de que é 100% trapaça.
+
+Meu amigo foi banido por trapacear! Ele nunca trapacearia! Vocês estão errados!
+--------------------------------------------------------------------------------------
+
+Pode ser difícil chegar a um acordo com o seu amigo, mas você terá que confiar na equipe. Peppy não está tentando estragar amizades, mas ele sabe que quase ninguém pode ser confiável. Mesmo aquelas pessoas que estão perto de você que você nunca poderia suspeitar podem agir atrás de você ou qualquer outra pessoa.
```

Comment:
- "peppy" isn't capitalized.
- The last sentence uses a lot of "você". Try something like "Mesmo aquelas pessoas que estão perto de você que nunca seriam suspeitas podem agir sem que qualquer pessoa saiba."